internal/app/machined/internal/phase/config/extra_disks.go

```diff
@@ -0,0 +1,106 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Renamed the file to `extra_disks.go` since `disks` is the word of choice in the rest of the code when talking about block devices.